SwiftUIHomeworks/SwiftUIHomeworks/Models/Product.swift

```diff
@@ -0,0 +1,97 @@
+// Product.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Furniture
+struct Product: Identifiable {
+    /// Id
+    let id = UUID()
+    /// Product title
+    let name: String
+    /// Image name
+    let imageName: String
+    /// Price
+    let price: Int
+    /// Discount price
+    let discountPrice: Int
+    /// Amount of product
+    var amount: Int
+
+    // MARK: - Public methods
+
+    static func getProducts() -> [Product] {
+        [
+            Product(
+                name: "Fanblades",
+                imageName: "fanblades",
```

Comment:
```suggestion
                imageName: "fanblades",
```
В константы все в этом файле